app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -415,7 +415,7 @@ function AcuantCapture(
    */
   function startCaptureOrTriggerUpload(event: MouseEvent) {
     if (event.target === inputRef.current) {
-      if (forceNativeCamera) {
+      if (forceNativeCamera && isMobile) {
```

Comment:
Checking mobile seems to be only indirectly related to the logic associated with starting Acuant capture. In fact, I hadn't realized previously that we're already evaluating whether to use Acuant on the lines immediately following this condition (`shouldStartAcuantCapture`), so the logic associated with forcing native camera in this condition seems a bit redundant and fragile. I think we should consider `forceNativeCamera` as part of that assignment and avoid the call to `forceUpload` altogether.

```tsx
    if (event.target === inputRef.current) {
      const isAcuantCaptureCapable = hasCapture && !acuantFailureCookie;
      const shouldStartAcuantCapture =
        isAcuantCaptureCapable &&
        capture !== 'user' &&
        !isForceUploading.current &&
        !forceNativeCamera;
      const shouldStartSelfieCapture =
        isAcuantLoaded && capture === 'user' && !isForceUploading.current;

      if (isAcuantCaptureCapable && forceNativeCamera) {
        trackEvent('IdV: Native camera forced after failed attempts', {
          field: name,
          failed_capture_attempts: failedCaptureAttempts,
          failed_submission_attempts: failedSubmissionAttempts,
        });
      }

      if (!allowUpload || shouldStartSelfieCapture || shouldStartAcuantCapture) {
        event.preventDefault();
      }

      // ...
```